src/ResourceManager/Sql/Commands.Sql/Data Sync/Model/AzureSqlSyncGroupModel.cs

```diff
@@ -0,0 +1,122 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Security;
+using Microsoft.Azure.Management.Sql.LegacySdk.Models;
+
+namespace Microsoft.Azure.Commands.Sql.DataSync.Model
+{
+    /// <summary>
+    /// Represents a sync group object 
+    /// </summary>
+    public class AzureSqlSyncGroupModel
+    {
+        /// <summary>
+        /// Gets or sets the resource Id
+        /// </summary>
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource group name
+        /// </summary>
+        public string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the server name
+        /// </summary>
+        public string ServerName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the database name
+        /// </summary>
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the sync group
+        /// </summary>
+        public string SyncGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sync database resource Id
+        /// </summary>
+        public string SyncDatabaseId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the frequency (in seconds) time of doing data synchronization
+        /// </summary>
+        public int? IntervalInSeconds { get; set; }
```

Comment:
If there is a server default this is appropriate.  In PowerShell, you can make this parameter optional and check BoundParameters  to see if the user provided it.  Limiting the number of values that the user has to provide is a good thing